templates/default/src/main.cairo

```diff
@@ -0,0 +1,5 @@
+func main(a : felt, b : felt) -> (res : felt):
+    let x = 'Hello World'
+    return (a+b)
+end
```

Comment:
I'd use Balance example from cairo docs.
https://www.cairo-lang.org/docs/hello_starknet/intro.html#your-first-contract

Balance contract seems to be the "Hello World" for blockchain technologies.